### ISSUE TITLE
Can't install package with wildcard version via apt

### ISSUE THREAD

**Issue by ghost** [ansible/ansible#13392]
##### Summary:

Everywhere it says that Ansible supports wildcards in package name and version yet I can't get it to work.
##### Steps To Reproduce:

```
- name: Install LLDP daemon package with apt
  apt: "name=lldpd=0.7* state=present" 
  become: true
  notify:
    - Restart lldpd service
```
##### Expected Results:

Installs 0.7.11-2 on jessie or 0.7.11-2~bpo70+1 on wheezy
##### Actual Results:

Host fails with:

> msg: pkgname wildcard and version can not be mixed

**Comment by chrrrles** [comment]
Adding as a bug report as this should already be fixed in 1.9.3:

https://github.com/ansible/ansible-modules-core/pull/323

**Comment by JeanFred** [comment]
Looks like since ansible/ansible-modules-core#323 was merged this can be closed? It does work for me with ansible 1.9.5
